### ISSUE TITLE
manual/scalaGuide/main/akka/ScalaAkka.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#990]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/akka/ScalaAkka.md
- diffs between [2.2.0/manual/scalaGuide/main/akka/ScalaAkka.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/akka/ScalaAkka.md) and [2.3.5/manual/scalaGuide/main/akka/ScalaAkka.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/akka/ScalaAkka.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/akka/ScalaAkka.md   2014-08-31 03:11:53.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/akka/ScalaAkka.md  2014-08-11 10:39:16.000000000 +0900
@@ -1,10 +1,11 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Integrating with Akka

 [Akka](http://akka.io/) uses the Actor Model to raise the abstraction level and provide a better platform to build correct concurrent and scalable applications. For fault-tolerance it adopts the ‘Let it crash’ model, which has been used with great success in the telecoms industry to build applications that self-heal - systems that never stop. Actors also provide the abstraction for transparent distribution and the basis for truly scalable and fault-tolerant applications.

 ## The application actor system

-Akka 2.0 can work with several containers called `ActorSystems`. An actor system manages the resources it is configured to use in order to run the actors which it contains. 
+Akka can work with several containers called `ActorSystems`. An actor system manages the resources it is configured to use in order to run the actors which it contains. 

 A Play application defines a special actor system to be used by the application. This actor system follows the application life-cycle and restarts automatically when the application restarts.

@@ -26,13 +27,13 @@

 > **Note:** You can also configure any other actor system from the same file; just provide a top configuration key.

-
+For Akka logging configuration, see [[configuring logging|SettingsLogger]].

 ## Scheduling asynchronous tasks

 You can schedule sending messages to actors and executing tasks (functions or `Runnable`). You will get a `Cancellable` back that you can call `cancel` on to cancel the execution of the scheduled operation.

-For example, to send a message to the `testActor` every 30 microseconds:
+For example, to send a message to the `testActor` every 300 microseconds:

 @[play-akka-actor-schedule-repeat](code/ScalaAkka.scala)


```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1113
